### PR TITLE
Document Bitnami services limitations and workarounds in restricted environments

### PR DESCRIPTION
**Pull request by gcapizzi**
# Which other branches do you want a technical writer to cherry-pick this PR to (if any)?

We would like the docs team to take a look at the document and decide how to appropriately present the different solutions proposed. We believe the first section should belong to the TAP 1.7 docs, and the second to the TAP 1.5 and 1.6 docs.

### CODE REVIEW

File: bitnami-services/upgrading-tkr.hbs.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    ```console
+    NAME            SYNCED   READY   COMPOSITION                                           AGE
+    mongodb-zfjr5   True     True    xmongodbinstances.bitnami.database.tanzu.vmware.com   3h24m
+    ```
+
+## <a id="tap-16"></a>If staying on TAP 1.6 or older
+
+Services claimed in TAP 1.6 won't be able to start on TKR 1.26.
+We recommend a workaround based on mutating pods via a mutating webhook. For example, using
+[Kyverno](https://kyverno.io/), the following `ClusterPolicy` will apply the necessary modifications to every
```
Comment (teddyking) [inline bitnami-services/upgrading-tkr.hbs.md:97]: **question** Is there any way we could limit this to only bitnami services pods?

File: bitnami-services/upgrading-tkr.hbs.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+    NAME                                                             REVISION   XR-KIND               XR-APIVERSION                                 AGE
+    ...
+    xmongodbinstances.bitnami.database.tanzu.vmware.com-734d138      4          XMongoDBInstance      bitnami.database.tanzu.vmware.com/v1alpha1    3h4m
+    xmongodbinstances.bitnami.database.tanzu.vmware.com-889eaeb      1          XMongoDBInstance      bitnami.database.tanzu.vmware.com/v1alpha1    3h29m
+    xmongodbinstances.bitnami.database.tanzu.vmware.com-d869e8c      2          XMongoDBInstance      bitnami.database.tanzu.vmware.com/v1alpha1    3h29m
+    xmongodbinstances.bitnami.database.tanzu.vmware.com-f1f3fe9      3          XMongoDBInstance      bitnami.database.tanzu.vmware.com/v1alpha1    3h5m
+    ...
+    ```
+
+    Take not of the name of the *highest* revision. In our example, this is revision 4 (`xmongodbinstances.bitnami.database.tanzu.vmware.com-734d138`).
```
Comment (teddyking) [inline bitnami-services/upgrading-tkr.hbs.md:46]:
```suggestion
    Take note of the name of the *highest* revision. In our example, this is revision 4 (`xmongodbinstances.bitnami.database.tanzu.vmware.com-734d138`).
```

File: bitnami-services/upgrading-tkr.hbs.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
+# Upgrading to TKR 1.26 or newer
+
+The newest version of TKR enforce a [`restricted` Pod Security
+Standard](https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted) for all pods
+running on the cluster. This will have an impact on your running Bitnami services. In this topic we explore
+possible solutions to this problem.
```
Comment (teddyking) [inline bitnami-services/upgrading-tkr.hbs.md:6]:
```suggestion
two possible solutions to this problem. Where possible, it is recommended to follow the first solution, which involves upgrading to TAP 1.7 and then updating CompositionRevision references for any existing Bitnami Services instances. See the "If upgrading to TAP 1.7" section below. If you do not wish or are unable to upgrade to TAP 1.7 for any reason, then an alternative solution is provided. See the "If staying on TAP 1.6 or older" section below.
```

File: bitnami-services/how-to-guides/index.hbs.md [hunk 4]
```diff
@@ -6,4 +6,5 @@ In this section:
 
 - [Configure private registry and VMware Application Catalog integration for Bitnami Services](configure-private-reg-integration.hbs.md)
 - [Obtain credentials for VMware Application Catalog integration](obtain-credentials-for-vac-integration.hbs.md)
+- [Upgrading to TKR 1.26 or newer](upgrading-tkr.hbs.md)
 - [Troubleshoot Bitnami Services](troubleshooting.hbs.md)
```
Comment (teddyking) [inline bitnami-services/how-to-guides/index.hbs.md:10]: **thought** Might also be good to add a quick section to the troubleshooting doc which links to the Upgrading to TKR doc.

**Review by teddyking — COMMENTED** [review]
I have given this a quick read through and left a few comments but generally LGTM.

File: bitnami-services/upgrading-tkr.hbs.md (hunk 3 above)
Comment (gcapizzi) [reply bitnami-services/upgrading-tkr.hbs.md:6]: Do we keep this document the same and backport it to every version, or do we have different documents for 1.7 and for 1.6 or older? I assume if someone is reading the 1.7 docs they're running on 1.7?

File: bitnami-services/how-to-guides/index.hbs.md (hunk 4 above)
Comment (gcapizzi) [reply bitnami-services/how-to-guides/index.hbs.md:10]: Done, please take a look!

File: bitnami-services/upgrading-tkr.hbs.md (hunk 1 above)
Comment (gcapizzi) [reply bitnami-services/upgrading-tkr.hbs.md:98]: I have added a label selector!

**Comment by gcapizzi** [comment]
Hey @snneji! As explained in the issue description, we were looking to split this PR in two - is that something you can help us with?

**Comment by snneji** [comment]
Sure @gcapizzi , I can do that when I review and edit it. I'll move the 1.6/1.5 related info from the new page from this version to those earlier versions. I might have a few more questions to ask as well.

**Pull request merged**